tests/storage/volume_metadata_test.py

```diff
@@ -355,6 +362,7 @@ def test_long_description(self, size):
         # POOL_UUID=
         # TYPE=PREALLOCATED
         # GEN=0
+        # SEQ=7
```

Comment:
In this test we *must* use the maximum value, so this should be:

    SEQ=4294967295

With this the test will tell us exactly how much space we have.

Reply:
ack